lib/idx/authenticator/WebauthnEnrollment.ts

```diff
@@ -0,0 +1,29 @@
+import { Authenticator } from './Authenticator';
+import { RemediationValues } from '../remediators';
+
+export interface WebauthnEnrollValues extends RemediationValues {
+  clientData?: string;
+  attestation?: string;
+}
+
+export class WebauthnEnrollment extends Authenticator<WebauthnEnrollValues> {
+  canVerify(values: WebauthnEnrollValues) {
+    const { clientData, attestation } = values;
+    return (!!clientData && !!attestation);
+  }
+
+  mapCredentials(values: WebauthnEnrollValues) {
+    const { clientData, attestation } = values;
+    return {
+      clientData,
+      attestation
+    };
+  }
+
+  getInputs() {
+    return [
+      { name: 'clientData', type: 'string', required: true, visible: false, label: 'Client Data' },
```

Comment:
What's `visible` for? `label` is for display purpose, but it also indicates this input is not visible, those two fields exist together looks contradict to me.
From the samples, looks like they are all hidden input, you can remove `label` fields from the metas.

Reply:
I took `visible` and `label` from actual IDX API response. 
Label can be potentially used to format error message if it's missing

Reply:
I see, maybe label means the field should be rendered, then we can use visible to control it will display or not.